File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/CSharp/1049/WFSequentialService/IWorkflow.cs
using System;
using System.Linq;
using System.Collections.Generic;
using System.Text;
using System.ServiceModel;

namespace $rootnamespace$
{
	// ПРИМЕЧАНИЕ. Команду "Переименовать" в меню "Рефакторинг" можно использовать для одновременного изменения имени интерфейса "$safeitemrootname$" в коде и файле конфигурации.
	[ServiceContract]
	public interface $safeitemrootname$
	{

		[OperationContract]
		string GetData(int value);

	}
}

File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/VisualBasic/1049/WFStateMachineService/IWorkflow.vb
Imports System.ServiceModel

' ПРИМЕЧАНИЕ. Можно использовать команду "Переименовать" в контекстном меню, чтобы изменить имя интерфейса "$safeitemrootname$" в коде и файле конфигурации.
<ServiceContract()> _
Public Interface $safeitemrootname$

    <OperationContract()> _
    Function GetData(ByVal value As Integer) As String

End Interface
